src/rock/models/feeds/resolver.js

```diff
@@ -43,6 +86,13 @@ export default {
         filterQueries.push(models.Like.getLikedContent(user._id, models.Node));
       }
 
+      //for news section in app -- all news
+      if (filters.includes("NEWS")) {
```

Comment:
I added this for the news section in the app. I talked with Sam and Rich, and thought the section should include all news, whereas the homefeed's is filtered by campus. Does that make sense? I couldn't think of a clean way of separating the two in the `content` filter. 

I then realized we're using the `content` query to pull news. Not the `userFeed` query. If we continue to do that, then I could remove this section.